composer.json

```
@@ -21,7 +21,7 @@
         "silverstripe/admin": "^1.7@dev",
         "silverstripe/versioned": "^1.7@dev",
         "silverstripe/versioned-admin": "^1.7@dev",
-        "silverstripe/graphql": "^3.4 || 4.x-dev",
+        "silverstripe/graphql": "^3.4 || ^4 || 4.x-dev",
```

Not sure what `|| 4.x-dev` is meant to add here ... `4.x-dev` should match the `^4` constraint.

```suggestion
        "silverstripe/graphql": "^3.4 || ^4",
```

---

True that, have updated